packages/nextjs/app/api/grants/[grantId]/route.ts

```diff
@@ -10,22 +10,31 @@ type ReqBody = {
   askAmount?: number;
   signature?: `0x${string}`;
   signer?: string;
+  private_note?: string;
 };
 
 export async function PATCH(req: NextRequest, { params }: { params: { grantId: string } }) {
   try {
     const { grantId } = params;
-    const { title, description, signature, signer, askAmount } = (await req.json()) as ReqBody;
+    const { title, description, signature, signer, askAmount, private_note } = (await req.json()) as ReqBody;
 
-    if (!title || !description || !askAmount || typeof askAmount !== "number" || !signature || !signer) {
+    if (
+      !title ||
+      !description ||
+      !askAmount ||
+      typeof askAmount !== "number" ||
+      !signature ||
+      !signer ||
+      !private_note
```

Comment:
I think we don't want this. Editing a grant without a private note is throwing this error.

d852b69aee53d8659c7f66b2abcc86a63f82303f

Reply:
Ohh yes ! Sorry completely missed it :( , Thanks for marking it and fixing !

Reply:
Ouch, my bad here!! I'll try to plan better the tests so don't miss basic use cases like that :/